### DISCUSSION TITLE
SASS Include Paths 9.3.0

### DISCUSSION THREAD

**Discussion by ivoilic** [vercel/next.js#10915] [Help]
Previously it was possible to include paths for sass like this:

```
const withSass = require('@zeit/next-sass');

module.exports = withSass({
  cssModules: true,
  sassLoaderOptions: {
    includePaths: ['./src/styles']
  }
});
```
How can that be done now with 9.3.0?

**Comment by timneutkens** [comment]
Landed support for sassOptions earlier today as an experimental flag: https://github.com/zeit/next.js/pull/11063
